src/NewCommand.php

```diff
@@ -162,16 +153,17 @@ protected function download($zipFile, $version = 'master')
      *
      * @param  string  $zipFile
      * @param  string  $directory
+     * @param  string  $filename
      * @return $this
      */
-    protected function extract($zipFile, $directory)
+    protected function extract($zipFile, $directory, $filename)
     {
         $archive = new ZipArchive;
 
         $response = $archive->open($zipFile, ZipArchive::CHECKCONS);
 
         if ($response === ZipArchive::ER_NOZIP) {
-            throw new RuntimeException('The zip file could not download. Verify that you are able to access: http://cabinet.laravel.com/latest.zip');
+            throw new RuntimeException("The zip file could not download. Verify that you are able to access: http://cabinet.laravel.com/{$filename}");
```

Comment:
Then here we are... the purpose to the pull request was to unify the way we render the file name to the user.